tests/app/services/test_process_request_json.py

```diff
@@ -162,21 +168,22 @@ def test_missing_field_in_transformation():
                 ]
             }
         },
-    ], TEXT_FIELDS_SET=set(['someField', 'otherField'])):
-        request = {
-            "otherField": ["wibble"],
-        }
+    ]
+    services_mapping.text_fields_set = set(['someField', 'otherField'])
+    request = {
+        "otherField": ["wibble"],
+    }
 
-        result = convert_request_json_into_index_json(request)
-        # really just checking this case doesn't throw!
+    result = convert_request_json_into_index_json(request)
+    # really just checking this case doesn't throw!
 
-        assert result == {
-            "otherField": ["wibble"],
-        }
+    assert result == {
```

Comment:
Adding that assertion is the work of about 20 seconds. It's not worthing arguing about really, so if you don't want to add it that's fine. To me it's just one of those small bonus things you can throw in at almost no cost.